components/search/SearchQueryOptions.vue

```diff
@@ -0,0 +1,434 @@
+<template>
+  <b-list-group
+    v-show="query && query.length > 0"
+    :id="elementId"
+    class="auto-suggest-dropdown"
+    data-qa="search suggestions"
+    role="listbox"
+    :aria-label="$t('searchSuggestions')"
+    :aria-hidden="!isActive"
+  >
+    <b-list-group-item
+      v-if="showLoader && isLoading"
+      class="loading"
+    >
+      {{ $t('loadingResults') }}{{ $t('formatting.ellipsis') }}
+    </b-list-group-item>
```

Comment:
This doesn't seem to be used anywhere. 'showLoader' defaults to 'false' and isn't  changed at any point.

Reply:
true, I've removed, good to simplify the component more